extensions/nginx/letsencrypt.js

```diff
@@ -43,10 +44,21 @@ module.exports = function letsencrypt(instance, email, staging, renew) {
 
         // This is an execa error
         if (error.stdout.match(/Skip/)) {
-            instance.ui.log('Certificate not due for renewal yet, skipping', 'yellow');
-            return;
+            // Certificate already exists
+            if (renew) {
+                instance.ui.log('Certificate not due for renewal yet, skipping', 'yellow');
```

Comment:
Can do this as a direct-to-master commit - I would like this to be "SSL certificate" to be consistent 😁